constellations/gemini-user/src/announcements/announce.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+require("./announce.sass");
+import React from "react";
+import API from "../utils/api.js";
+import { keys, sortBy, groupBy } from "lodash";
+
+export class DateList extends React.Component {
+    render() {
+        let a = this.props.announcements;
+        let date = this.props.postedAt;
+        date = date.substring(0, 10);
+        date = date.replace(/-/g, "/");
+        date = date.substr(5) + "/" + date.substring(0, 4);
```

Comment:
ok nice! To handle dates and time, we will use a Javascript library called moment.

```
import moment from "moment";
...

var postedAt = moment(this.props.postedAt); // creates a "moment" object from a Date
var day = postedAt.day();                                   // extracts the "day" out of the date
console.log(postedAt.toString());                      // prints the Date as a string.
```
Look at the docs here (https://momentjs.com/docs/#/get-set/) to get more info about better handling dates.